build.sbt

```diff
@@ -33,7 +33,9 @@ scalacOptions := Seq(
   "-encoding",
   "utf8",
   "-feature",
-  "-language:reflectiveCalls"
+  "-language:reflectiveCalls",
+  "-Yrangepos",
+  "-Ywarn-unused-import"
```

Comment:
I think we should use the full set of scalac flags [proposed by **tpolecat**](https://tpolecat.github.io/2017/04/25/scalac-flags.html).

There is even an SBT _plugin_ for it.

Reply:
I added those because they are required by scalafix, but yes we could also add some of those recommended by Rob.